scoring.js

```diff
@@ -48,6 +48,7 @@ module.exports.calculateScore = function (player) {
     case 'TE':
       return calculateReceivingScore(player)
     default:
-      return 0
+      throw new Error('Unknown player position (K)')
```

Comment:
While this will work for the specific position of K, it will fail for any other unknown position. If we called this function with a player who had a position of LB we would be told `Unknown player position (K)` which would be a very unhelpful error message.